src/index.test.tsx

```diff
@@ -71,7 +71,7 @@ function TestComponent(props) {
     <div className="test-component">
       <div className="clear-text" onClick={clearText} />
       <div className="load-more" onClick={loadMore} />
-      <div className="accept-suggestion" onClick={acceptSuggestion} />
+      <div className="accept-suggestion" onClick={(text: any) => acceptSuggestion(text)} />
```

Comment:
Should this take an event instead of text?

Reply:
the problem is that `acceptSuggestion` has type `string => void`, so if you just pass `acceptSuggestion` to `onClick` it throws a type error because `onClick` will not receive an argument of type `string` but of type `MouseEvent`, so it says 'hey, `acceptSuggestion` isn't going to get the type it expects!'

anyhow, I have this `text: any => void` function here to basically scrub typing on this function to avoid the type error - I'm only using this component as basically a little test harness anyhow, so I'm not worried about typing everything correctly here. It's just basically so I can have the `useCourseSearch` hook mounted in a component and then get access to callbacks like `acceptSuggestion` via enzyme's `.props` method in order to test them.

so anyhow it's a bit convoluted but there's good reason for it! haha